src/issueFinder.ts

```diff
@@ -56,11 +58,20 @@ export class IssueFinder {
             createdAt: node.createdAt,
             repositoryNameWithOwner: node.repository.nameWithOwner,
             url: node.url,
+            updatedAt: node.createdAt,
```

Comment:
I'm thinking this should be node.updatedAt

Reply:
Whoops. Nice catch.